src/index.js

```diff
@@ -243,13 +252,7 @@ class DropdownTreeSelect extends Component {
         this.handleClick()
       }
       return
-    } else if (
-      e.key === 'Backspace' &&
-      tags &&
-      tags.length &&
-      this.searchInput &&
-      this.searchInput.value.length === 0
-    ) {
+    } else if (e.key === 'Backspace' && tags.length && this.searchInput.value.length === 0) {
       const lastTag = tags.pop()
       this.onCheckboxChange(lastTag._id, false)
     } else {
```

Comment:
What does this empty return accomplish? What'll happen if we removed it?

Reply:
It will avoid preventDefault to be triggered in cases where it did not match any if statement.